dropwizard-validation/src/test/java/io/dropwizard/validation/PortRangeValidatorTest.java

```diff
@@ -80,4 +83,12 @@ void rejectsInvalidPortsInList() {
         assertThat(ConstraintViolations.format(validator.validate(example)))
             .containsOnly("ports[0].<list element> must be between 1 and 65535");
     }
+
+    @Test
+    @SuppressWarnings("NullAway")
+    void rejectsNull() {
```

Comment:
Shouldn't this method be named `acceptsNull`?

Reply:
That was a leftover of https://github.com/dropwizard/dropwizard/pull/5689#discussion_r939708200. 😓

Reply:
Okay, thanks :+1: